railties/test/commands/log_test.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+require "rails/commands/log/log_command"
+
+class Rails::Command::LogTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+  setup :build_app
+  teardown :teardown_app
+
+  test "`rails log:clear` clears all environments log files by default" do
+    Dir.chdir(app_path) do
+      File.open("config/environments/staging.rb", "w")
+
+      File.write("log/staging.log", "staging")
+      File.write("log/test.log", "test")
+      File.write("log/dummy.log", "dummy")
+
+      rails "log:clear"
+
+      assert_equal 0, File.size("log/test.log")
+      assert_equal 0, File.size("log/staging.log")
+      assert_equal 5, File.size("log/dummy.log")
```

Comment:
I realize this is also copied from the original test, but asserting the exact file size seems overly specific to me.

We could use `assert_not_equal 0`, or we could add private `assert_file_empty` and `assert_file_not_empty` helpers that incorporate `app_path` (in order to drop the `Dir.chdir` block).  e.g.:

```ruby
assert_file_not_empty "log/dummy.log"
```

Reply:
I've added assertion for `assert_file_empty` and `assert_file_not_empty`